tools/pipelines/test-dds-stress.yml

```diff
@@ -59,9 +59,52 @@ parameters:
     testFileTarName: matrix
     testCommand: test:stress
 
+
 stages:
   - template: templates/include-conditionally-run-stress-tests.yml
     parameters:
       artifactBuildId: $(resources.pipeline.client.runID)
       packages: ${{ parameters.packages }}
       testWorkspace: ${{ variables.testWorkspace }}
+      stressMode: ${{ parameters.stressMode }}
+
+  # Capture telemetry about pipeline stages
+  - stage: upload_run_telemetry
```

Comment:
looks like the extra changes got brought back here. I'm approving, but revert these please :)